Sources/NIO/EventLoop.swift

```diff
@@ -127,6 +127,32 @@ public final class RepeatedTask {
         self.reschedule()
     }
 }
+
+/// An iterator over an array of 'EventLoop's.
```

Comment:
that we're iterating over an _array_ is an implementation detail. Maybe write something like

> An iterator over the `EventLoop`s forming an `EventLoopGroup`.

or something.